docs_versioned_docs/version-ros2jazzy/ros/tutorials/simulator/install.mdx

```diff
@@ -16,15 +16,24 @@ It is highly recommended to install the simulator on a computer with a dedicated
 
 Follow the [Offboard Computer](../../installation/offboard_pc.mdx) setup guide.
 
-### Installing Gazebo Fortress
+### Installing Gazebo Harmonic
+
+:::note
+
+See Gazebo's [installation instructions](https://gazebosim.org/docs/harmonic/install_ubuntu/) for
+more information on installing and uninstalling Gazebo Harmonic.
+
+:::
+
 
 Run the following commands in your terminal:
 
 ```
-sudo apt-get update && sudo apt-get install wget
-sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
-wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
-sudo apt-get update && sudo apt-get install ignition-fortress
+sudo apt-get install curl lsb-release gnupg
```

Comment:
Do we need this with the Gazebo vendor package?

Reply:
Good catch. You're correct: we don't this.